addons/hr_holidays/tests/test_allocations.py

```diff
@@ -402,3 +403,24 @@ def test_create_allocation_from_company_with_no_employee_for_current_user(self):
         allocation_form.holiday_status_id = self.leave_type
         allocation = allocation_form.save()
         self.assertTrue(allocation)
+
+    def test_allocation_request_with_date_from(self):
+        with self.assertRaises(AssertionError):
+            with Form(self.env['hr.leave.allocation'].with_user(self.user_hrmanager), 'hr_holidays.hr_leave_allocation_view_form') as allocation_form:
```

Comment:
```suggestion
        allocation = self.env['hr.leave.allocation'].with_user(self.user_hrmanager)
        allocation_view = 'hr_holidays.hr_leave_allocation_view_form'
        
        with self.assertRaises(AssertionError):
            with Form(allocation, allocation_view) as allocation_form:
```
You should use variable for Form arguments; apply same in below.